opentracing/scope.py

```diff
@@ -73,5 +73,10 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         """Calls :meth:`close()` when the execution is outside the Python
         Context Manager.
+
+        If exception has occurred during execution, it is automatically logged
+        and added as a tag to the :class:`Span`.
+        :attr:`~operation.ext.tags.ERROR` will also be set to `True`.
```

Comment:
Please use [numpy syntax](https://github.com/aws/aws-sam-cli/blob/346ff2923477d8bc0f4fffb5da3629bcfc935b96/DEVELOPMENT_GUIDE.md#code-conventions) for docstrings.